crates/glue/src/CGlue.roc

```diff
@@ -0,0 +1,31 @@
+app [makeGlue] { pf: platform "../platform/main.roc" }
+
+import pf.Types exposing [Types]
+# import pf.Shape exposing [Shape, RocFn]
+import pf.File exposing [File]
+# import pf.TypeId exposing [TypeId]
+
+makeGlue : List Types -> Result (List File) Str
```

Comment:
Can you add a doc domment (using `## `)  to this function?

Reply:
Sure!